### PR TITLE
MedicationDispense transformer first iteration

### PR DESCRIPTION
**Pull request by treynolds-va**
First iteration of building MedicationDispense transformer

### CODE REVIEW

**Comment by bschofield-va** [comment]
@jnmiller-va might want to hold off on more changes

File: vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/condition/ConditionId.java [hunk 1]
```diff
@@ -19,6 +19,9 @@
   ConditionId(String privateId) {
     var colon = privateId.indexOf(':');
     if (colon == -1 || colon == privateId.length() - 1) {
+      /*
```
[the remark this answers is not in the document]
Comment (bschofield-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/condition/ConditionId.java:22]: yes

**Comment by jnmiller-va** [comment]
> @jnmiller-va might want to hold off on more changes

Will do!

**Comment by bschofield-va** [comment]
@jnmiller-va sorry... we tend to operate under the assumption that once the PR is open we'll consider it complete from your perspective and ready for us to take over.

**Comment by jnmiller-va** [comment]
> @jnmiller-va sorry... we tend to operate under the assumption that once the PR is open we'll consider it complete from your perspective and ready for us to take over.

Yup. That was our fault. Thought we were further along. Got ahead of ourselves.

**Review by bschofield-va — APPROVED** [review]

**Pull request merged**